### PR TITLE
fix(integration):stripe-app access token refresh on expire

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes
Nango was finding it hard to refresh `stripe-app`  access token upon expiration. This was because upon exchanging a code for an access token, there is no `expire_at` or `expire_in` value in the response. However, according to the [docs](https://docs.stripe.com/stripe-apps/api-authentication/oauth#refresh-access-token), the access token expires after an hour, which is equivalent to 3600 seconds. I added that value during the creation of the Stripe app token. This will help facilitate automatic refresh on access token expiration.

### CODE REVIEW

File: packages/shared/lib/clients/provider.client.ts [hunk 1]
```diff
@@ -192,6 +200,70 @@ class ProviderClient {
...
+            if (response.status === 200 && response.data) {
+                return {
+                    access_token: response.data['access_token'],
+                    livemode: response.data['livemode'],
+                    refresh_token: response.data['refresh_token'],
+                    scope: response.data['scope'],
+                    stripe_publishable_key: response.data['stripe_publishable_key'],
+                    stripe_user_id: response.data['stripe_user_id'],
+                    token_type: response.data['token_type'],
+                    expires_in: 3600
```
Comment (khaliqgant) [inline packages/shared/lib/clients/provider.client.ts]: Can we make this a const and put this at the top of the file in the event it ever changes we can easily change it?

File: packages/shared/lib/clients/provider.client.ts [hunk 2]
```diff
@@ -192,6 +200,70 @@ class ProviderClient {
...
```
Comment (khaliqgant) [inline packages/shared/lib/clients/provider.client.ts]: use const as mentioned below

File: packages/shared/lib/clients/provider.client.ts (hunk 1 above)
Comment (hassan254-prog) [reply packages/shared/lib/clients/provider.client.ts]: Sure, I'll refactor both methods by placing `const expires_in` at the top of the file. This way, it will be easier to locate and modify if needed in the future. Thank you.

File: packages/shared/lib/clients/provider.client.ts [hunk 3]
```diff
@@ -217,7 +290,7 @@ class ProviderClient {
                 };
             }
 
-            throw new NangoError('tiktok_token_request_error', response.data);
+            throw new NangoError('tiktok_token_request_error');
```
Comment (khaliqgant) [inline packages/shared/lib/clients/provider.client.ts]: Why this change? Seems unrelated

Comment (hassan254-prog) [reply packages/shared/lib/clients/provider.client.ts]: Reverted the change.
